modules/client/src/internal/client/methods.ts

```diff
@@ -300,7 +346,7 @@ export class ClientMethods extends ClientCore implements IClientMethods {
       // this is a temporal fix and should be changed by the line above
       // but the current daos in subgraph dont have a proper metadata
       const stringMetadata = await this.ipfs.fetchString(
-        "QmebY8BGzWBUyVqZFMaFkFmz3JrfaDoFP5orDqzJ1uiEkr",
+        "QmebY8BGzWBUyVqZFMaFkFmz3JrfaDoFP5orDqzJ1uiEkr"
```

Comment:
Should we be using the real data from subgraph or not yet? We need a follow-up task if not.

Reply:
I don't think so, because the subgraph is still not deployed and tests are failing if I use it